### PR TITLE
ex1 change values sign inside numpy array.

### PR DESCRIPTION
**Pull request by YanivSonino**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+.idea_modules/
+
+# JIRA plugin
+atlassian-ide-plugin.xml
+
+# Crashlytics plugin (for Android Studio and IntelliJ)
+com_crashlytics_export_strings.xml
+crashlytics.properties
+crashlytics-build.properties
+fabric.properties
```
Comment (mattisevitt) [inline .gitignore:152]: Why is this here?

**Review by mattisevitt — CHANGES_REQUESTED** [review]

**Review by talbarfm — APPROVED** [review]
